### PR TITLE
Fix nerdctl stats output

### PR DESCRIPTION
**Pull request by fahedouch**
thanks to @liubin  to detect the issue (https://github.com/containerd/nerdctl/issues/1278#issuecomment-1205151926)

fix stats output and ensure error display (https://github.com/containerd/nerdctl/issues/1278)

Signed-off-by: fahed dorgaa <fahed.dorgaa@gmail.com>

### CODE REVIEW

File: cmd/nerdctl/stats.go [hunk 1]
```diff
@@ -283,17 +283,6 @@ func statsAction(cmd *cobra.Command, args []string) error {
 		// make sure each container get at least one valid stat data
 		waitFirst.Wait()
 
-		var errs []string
```
Comment (manugupt1) [inline cmd/nerdctl/stats.go:286]: Why remove this block?

Comment (fahedouch) [reply cmd/nerdctl/stats.go:286]: useless regarding this https://github.com/containerd/nerdctl/pull/1298/files#diff-8541fa08e70d092bb039071593918dac552c3cbf1088d15e71d0f6117ad4b249R306

the error display should be within the ticker

File: cmd/nerdctl/stats_linux.go [hunk 2]
```diff
@@ -62,11 +62,13 @@ func setContainerStatsAndRenderStatsEntry(previousStats *statsutil.ContainerStat
 		if err != nil {
 			return statsutil.StatsEntry{}, fmt.Errorf("failed to retrieve the statistics in netns %s: %v", ns, err)
 		}
+		defer ns.Close()
```
Comment (manugupt1) [inline cmd/nerdctl/stats_linux.go:65]: Can we wrap this into a function and log errors if Close returns an error; that would help us catch file leaks if there are any if still remaining? Ideally, this should never happen.

**Review by junnplus — APPROVED** [review]
Thanks, LGTM

**Review by manugupt1 — APPROVED** [review]

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**